unlock-app/src/middlewares/storageMiddleware.js

```diff
@@ -297,6 +309,13 @@ const storageMiddleware = config => {
           storageService.getKeyPrice(lock.address)
         }
 
+        if (action.type === SIGN_METADATA_RESPONSE) {
+          const { signature, keyIds, lockAddress } = action
+          keyIds.forEach(id => {
```

Comment:
Do you mind adding a quick TODO that at some point we should able to retrieve all the keyys at once?